setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+VERSION = 1
+
+setup(
```

Comment:
Yeah, except for the fact we're not writing a `setup.py` for the sake of writing one, we're writing one because dh-virtualenv required it.